polus-vector-label-plugin/src/main.py

```diff
@@ -0,0 +1,153 @@
+from bfio import  BioWriter , OmeXml
+import argparse, logging, sys
+import numpy as np
+from pathlib import Path
+import re
+import zarr
+import mask
+from numba import jit
+
+
+@jit(nopython=True)
+def _label_overlap(x, y):
+    """ fast function to get pixel overlaps between masks in x and y
+    Args:
+    x(array[int]): ND-array where 0=NO masks; 1,2... are mask labels
+    y(array[int]): ND-array where 0=NO masks; 1,2... are mask labels
+    Returns:
+    overlap(array[int]): ND-array.matrix of pixel overlaps of size [x.max()+1, y.max()+1]
+
+    """
```

Comment:
All values under a section should be tabbed as follows:
```python
    Args:
        x(array[int]): ND-array where 0=NO masks; 1,2... are mask labels
        y(array[int]): ND-array where 0=NO masks; 1,2... are mask labels
    Returns:
        overlap(array[int]): ND-array.matrix of pixel overlaps of size [x.max()+1, y.max()+1]
```

Do this for all docstrings.